dm/tests/online_ddl/run.sh

```diff
@@ -29,6 +20,27 @@ function run() {
 	sed -i "/relay-binlog-name/i\relay-dir: $WORK_DIR/worker2/relay_log" $WORK_DIR/source2.yaml
 	dmctl_operate_source create $WORK_DIR/source1.yaml $SOURCE_ID1
 
+	cp $cur/conf/check-task.yaml $WORK_DIR/check-task.yaml
+	# prevent start task in process of online-ddl
+	run_sql_file $cur/data/check.gho.db1.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"check-task $WORK_DIR/check-task.yaml" \
+		"your ddl is in pt\/ghost online-ddl" 1
+
+	run_sql_file $cur/data/check.pt.db1.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"check-task $WORK_DIR/check-task.yaml" \
+		"your ddl is in pt\/ghost online-ddl" 1
```

Comment:
maybe add one positive case.

Reply:
all `start-task` command will `check-task` first. So that we `start-task` success means `check-task` success.